docs/rfc/rfc-100-abci-vote-extension-propag.md

```diff
@@ -426,36 +428,36 @@ extensions will *always* be available when calling `PrepareProposal` at height *
 This level of guarantees is probably not strong enough for vote extensions to be useful for some
 important use cases that motivated them in the first place, e.g., encrypted mempool transactions.
 
-Solution 1, while being simple in that the changes needed in the current Tendermint codebase would
+Solution 1, while being simple in that the changes needed in the current CometBFT codebase would
 be rather small, is changing the block format, and would therefore require all blockchains using
-Tendermint v0.35 or earlier to hard-fork when upgrading to v0.36.
+ABCI 1.0 or earlier to hard-fork when upgrading to ABCI 2.0.
 
 Since Solution 2 can be attacked, one might prefer Solution 3, even if it is more involved
 to implement. Further, we must elaborate on how we can turn Solution 3, described in abstract
 terms in the previous section, into a concrete implementation compatible with the current
-Tendermint codebase.
+CometBFT codebase.
 
 ### Current Limitations and Possible Implementations
 
-The main limitations affecting the current version of Tendermint are the following.
+The main limitations affecting the current version of CometBFT are the following.
 
 - The current version of the blocksync reactor does not use the full
-  [light client verification](https://github.com/tendermint/tendermint/blob/4743a7ad0/spec/light-client/README.md)
+  [light client verification][light-client-spec]
   algorithm to validate blocks coming from other peers.
 - The code being structured into the blocksync and consensus reactors, only switching from the
   blocksync reactor to the consensus reactor is supported; switching in the opposite direction is
   not supported. Alternatively, the consensus reactor could have a mechanism allowing a late node
   to catch up by skipping calls to `PrepareProposal`/`ProcessProposal`, and
   `ExtendVote`/`VerifyVoteExtension` and only calling `FinalizeBlock` for each height.
-  Such a mechanism does not exist at the time of writing this RFC.
+  Such a mechanism does not exist at the time of writing this RFC (2023-03-02).
 
-The blocksync reactor featuring light client verification is being actively worked on (tentatively
-for v0.37). So it is best if this RFC does not try to delve into that problem, but just makes sure
+The blocksync reactor featuring light client verification is among the CometBFT team' current priorities.
```

Comment:
This was what I worked on last year, do we still plan to go ahead with this ?

Reply:
Good question :-)
I've just checked and I can't see any issue in CometBFT on this. IIRC it's because we only migrated those issues we'd need for Q1.
An idea would be to go back to the Tendermint Core repo, dig up the set of issues/PRs related to this, and write a tracking issue containing all that info. Then, we could use that issue to prioritize w.r.t. to other things we have on our plate.